### ISSUE TITLE
error: unable to read module map contents from 'Target Support Files/SDWebImage/SDWebImage.modulemap': Error Domain=NSCocoaErrorDomain Code=260 "The file “SDWebImage.modulemap” couldn’t be opened because there is no such file."

### ISSUE THREAD

**Issue by TheNotoriousCoder** [rs/SDWebImage#1955]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios 
 Platform Version        | e.g. 10.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. cocoapods 
 Xcode Version           | e.g. Xcode 8.3.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Hello I have a constant issue with this error. This error has a mind of it's own. It will either occur or not occur. As soon as I open and run/build the project determines if I come across the error.

I have done the following, to no success, in order to get rid of this error:

Delete/re-installed my cocoapods
Deleted the "Build" folder
I have done the following:

Open a terminal app
cd ~/Library/Developer/Xcode/DerivedData
rm -rf Build/* see below if you have a customized build location
rm -rf ModuleCache/*
Also done:

Clean Build Folder
Done everything I have been advised to do, yet cannot seem to fix this issue. I even went as far as creating a Target Support Files folder in the directory of where all my.swift/cocoatouch classes are however. I am still unsuccessful.

I have no more idea on what to do.

Could someone please help?

The only way in which I am combating this problem without fixing it is closing the project and reopening it, hoping for the error not to appear.

I think this also has an affect on my app, as it'll work fine for a few days then all of a sudden I'll open the app and it will constantly crash.

What could possibly be the issue?

**Comment by bpoplauschi** [comment]
This is a CocoaPods installation issue, you should look for similar issues in their own repo.

**Comment by scott-lydon** [comment]
I'm having this issue while my teammate is creating a pod and my pod will be installing a using his pod.  If it is on the side of the person making the pod to be consumed by the other pod. What can that person do about this?
